### PR TITLE
Add README and Licence file

### PR DESCRIPTION
**Pull request by Linary**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
 # hugegraph-hubble
+
+hugegraph-hubble A graph management and analysis platform that provides features: graph data load, schema management, graph relationship analysis and graphical display, and more.
```
Comment (zhoney) [inline README.md]: hugegraph-hubble is a graph management

Comment (zhoney) [inline README.md]: graph relationship analysis and graphical display.

deleting ", and more" is ok

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+## Features
+
+- Graph connection management, which allows users to easily switch graph to operate
+- Graph data load, supporting load large amounts of data from file into hugegraph-server
```
Comment (zhoney) [inline README.md]: choose one style for:
```
which allows
supporting
provides
user can
```

Comment (zhoney) [inline README.md]: files

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+- Schema management, provides an entrence to easily perform schema manipulation and display
```
Comment (zhoney) [inline README.md]: entrance

File: hubble-be/src/main/java/com/baidu/hugegraph/service/GremlinQueryService.java [hunk 4]
```diff
@@ -240,13 +243,13 @@ private ExecutePlan explain(String gremlin) {
         String explain = gremlin + ".explain()";
         ResultSet resultSet = client.gremlin().gremlin(explain).execute();
         if (resultSet.data().size() != 1) {
-            throw new InternalException("生成执行计划失败");
+            throw new InternalException("Generate execution plan failed");
```
Comment (zhoney) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/GremlinQueryService.java]: Failed to generate execution plan

File: hubble-be/src/main/java/com/baidu/hugegraph/service/GremlinQueryService.java [hunk 5]
```diff
@@ -240,13 +243,13 @@ private ExecutePlan explain(String gremlin) {
...
         }
         Map<String, Object> steps = (Map<String, Object>) resultSet.data()
                                                                    .get(0);
         List<String> finalSteps = (List<String>) steps.get("final");
         if (finalSteps.size() == 0) {
-            throw new InternalException("执行计划格式不正确");
+            throw new InternalException("Execution plan format is incorrect");
```
Comment (zhoney) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/GremlinQueryService.java]: Incorrect execution plan format

File: hubble-be/src/main/java/com/baidu/hugegraph/controller/GremlinQueryController.java [hunk 6]
```diff
@@ -46,7 +46,7 @@ public GremlinResult execute(@RequestBody GremlinQuery query) {
 
     private void checkParamsValid(GremlinQuery query) {
         Ex.check(!StringUtils.isEmpty(query.getContent()),
-                 "Gremlin语句不能为null或空");
+                 "Gremlin sentence can't be null or empty");
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/controller/GremlinQueryController.java]: sentence => statement

File: README.md [hunk 7]
```diff
@@ -1 +1,18 @@
 # hugegraph-hubble
+
```
Comment (javeme) [inline README.md]: add badges

**Pull request closed without merging**